addons/prometheus/prometheus.yaml

```diff
@@ -10,11 +10,11 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.38.1-28"
-    appversion.kubeaddons.mesosphere.io/prometheus-operator: "0.38.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.44.0-1"
+    appversion.kubeaddons.mesosphere.io/prometheus-operator: "0.44.0"
     appversion.kubeaddons.mesosphere.io/prometheus: "2.19.2"
-    appversion.kubeaddons.mesosphere.io/alertmanager: "0.20.0"
-    appversion.kubeaddons.mesosphere.io/grafana: "6.7.3"
+    appversion.kubeaddons.mesosphere.io/alertmanager: "0.21.0"
+    appversion.kubeaddons.mesosphere.io/grafana: "6.1.17"
```

Comment:
weird. It looks like we downgraded but we must have had the wrong version of grafana before
![image](https://user-images.githubusercontent.com/5897740/105100907-e3330a80-5a62-11eb-9fe1-a35dbc62ef38.png)

Reply:
yeah im not sure what that is about, but i assumed we never had the correct version?